TODO.txt

```diff
@@ -27,8 +20,6 @@ Other
 -----
 * Remove legacy pretty printing workaround for ``pytest`` in ``conftest.py``
   once minimal required ``numpy`` is set to >= 1.14.0.
-* Remove deprecated ``Hxx, Hxy, Hyy`` API of ``hessian_matrix_eigvals`` in
```

Comment:
This was planned for 0.17 so why do it now?

Reply:
I think maybe it was a misclassification in the TODO list.

The warning states 0.16
https://github.com/scikit-image/scikit-image/pull/3851/files#diff-70ef54b6b626ce40eefa32dae16fc130L338

Reply:
@emmanuelle actually, it was in `Other` section.

Originally, the warning was introduced here -https://github.com/scikit-image/scikit-image/commit/ef07903e71aa2e4d2968600e3cdd41b4078b040b (https://github.com/scikit-image/scikit-image/pull/2854), before 0.14.0. It looks like there is no mistake, and the change was correctly scheduled for 0.16.0 (2 major releases: deprecation in 0.15, removal in 0.16).